sdk/nodejs/errors.ts

```diff
@@ -81,3 +81,28 @@ export function isGrpcError(err: Error): boolean {
     const code = (<any>err).code;
     return code === grpc.status.UNAVAILABLE || code === grpc.status.CANCELLED;
 }
+
+export class PropertyError {
```

Comment:
Based on the name and its properties, I feel like I want be able to throw this rather than `InvalidInputPropertiesError`, but that doesn't seem to be the intention of this design.

Reply:
I would support discussing this in ideation too -- I don't think this as idiomatic as it could be.